exporters/zipkin/model_test.go

```diff
@@ -409,7 +410,7 @@ func TestModelConversion(t *testing.T) {
 				"attr1":            "42",
 				"attr2":            "bar",
 				"attr3":            "[0,1,2]",
-				"otel.status_code": "Error",
+				"otel.status_code": "ERROR",
```

Comment:
We only ever have tests where the status code is `ERROR`. How about adding two, for `OK` and `UNSET`?

Reply:
for sure, will add corresponding test cases.

Reply:
added